commands/generateTypes.go

```diff
@@ -0,0 +1,73 @@
+package commands
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	versionInfo "github.com/StackExchange/dnscontrol/v3/pkg/version"
+	"github.com/urfave/cli/v2"
+)
+
+var _ = cmd(catUtils, func() *cli.Command {
+	var args TypesArgs
+	return &cli.Command{
+		Name:  "types",
```

Comment:
I'm not in love with having the subcommand called "types". However if that's what other similar systems use, I'd rather be consistent with them.  Could the name refer to typescript (or ts) to be clear what it is for?

Reply:
The key in the JS `package.json` is `types` (which I kind of based it on). I kind of shy away from using `typescript` since I want it to be clear that this doesn’t actually *run* TypeScript, it just emits the file. Maybe `dnscontrol export-ts-types` or `emit-ts-types` or something?

Reply:
How about `write-types`?  Its short and easy(ish) to remember.